.github/workflows/_sitrep_mgmn.yaml

```diff
@@ -0,0 +1,116 @@
+name: ~Generate sitrep for Multi-Node Multi-GPU tests
+
+on:
+  workflow_call:
+    inputs:
+      BADGE_FILENAME:
+        type: string
+        description: 'Name of the endpoint JSON file for shields.io badge'
+        required: true
+      ARTIFACT_NAME:
+        type: string
+        description: 'Name of the artifact zip file'
+        required: true
+    outputs:
+      STATUS:
+        description: 'Summary pass/fail value indicating if results from tests are acceptable'
+        value: ${{ jobs.sitrep.outputs.STATUS }}
+
+jobs:
+  sitrep:
+    runs-on: ubuntu-22.04
+    env:
+      BADGE_FILENAME_FULL: ${{ inputs.BADGE_FILENAME }}.json
+    outputs:
+      STATUS: ${{ steps.gen-sitrep.outputs.STATUS }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Download all artifacts from the previous jobs
+        uses: actions/download-artifact@v3
+
+      - name: Write Test Status
+        shell: bash -x -e {0}
+        run: |
+          EXIT_STATUSES="${GITHUB_RUN_ID}-*/*-status.json"
+
+          cat <<EOF >>$GITHUB_STEP_SUMMARY
```

Comment:
Please use a custom variable to pass information between job steps. Our plan is to collectively create a workflow run summary using a 'finalization' job so that this summary is generated as a single piece of Markdown document. Otherwise, we cannot control the order in which the individual step summaries show up in the workflow summary and the result will be fragmented.

Reply:
Done.